api/src/test/kotlin/edu/wgu/osmt/io/xlsx/RichSkillXlsxExportTest.kt

```diff
@@ -0,0 +1,81 @@
+package edu.wgu.osmt.io.xlsx
+
+import edu.wgu.osmt.SpringTest
+import edu.wgu.osmt.collection.Collection
+import edu.wgu.osmt.config.AppConfig
+import edu.wgu.osmt.db.PublishStatus
+import edu.wgu.osmt.keyword.Keyword
+import edu.wgu.osmt.keyword.KeywordTypeEnum
+import edu.wgu.osmt.richskill.RichSkillAndCollections
+import edu.wgu.osmt.richskill.RichSkillDescriptor
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import java.time.LocalDateTime
+import java.util.UUID
+
+internal class RichSkillXlsxExportTest @Autowired constructor(
+    appConfig: AppConfig
+) : SpringTest() {
+
+    val richSkillXlsExport = RichSkillXlsxExport(appConfig)
+
+    val collection = Collection(
+        id = 123,
+        creationDate = LocalDateTime.now(),
+        name = "name",
+        status = PublishStatus.Draft,
+        updateDate = LocalDateTime.now(),
+        uuid = UUID.randomUUID().toString()
+    )
+
+    @Test
+    fun`columnTranslations() should retrieve only 14 columns if no Alignments are present`() {
+        //Arrange
+        val rsd =  RichSkillDescriptor(
+            id = 123,
+            creationDate = LocalDateTime.now(),
+            name = "name",
+            statement = "statement",
+            updateDate = LocalDateTime.now(),
+            uuid = UUID.randomUUID().toString()
+        )
+
+        //Act
+        val result = richSkillXlsExport.columnTranslations(listOf(RichSkillAndCollections(rsd, setOf(collection))))
+
+        //Assert
+        Assertions.assertThat(result).hasSize(14)
+    }
+
+    @Test
+    fun`columnTranslations() should retrieve 17 columns if Alignments are present`() {
+        //Arrange
+        val rsd =  RichSkillDescriptor(
+            id = 123,
+            creationDate = LocalDateTime.now(),
+            keywords = listOf(
+                Keyword(
+                    id = 123,
+                    creationDate = LocalDateTime.now(),
+                    type = KeywordTypeEnum.Alignment,
+                    updateDate = LocalDateTime.now()
+                )),
+            name = "name",
+            statement = "statement",
+            updateDate = LocalDateTime.now(),
+            uuid = UUID.randomUUID().toString()
+        )
+
+        //Act
+        val result = richSkillXlsExport.columnTranslations(listOf(RichSkillAndCollections(rsd, setOf(collection))))
+
+        //Assert
+        Assertions.assertThat(result).hasSize(17)
+    }
+
+    @Test
+    fun`toXls() should convert columns to xls object without Alignments`() {
```

Comment:
we haven't completed this test, I recall I started it, but it should be changed to check if the result is a ByteArray object.. something like: 

```
@Test
    fun`toXls() should convert columns to byteArray object`() {
        //Arrange
        val rsd =  RichSkillDescriptor(
            id = 123,
            creationDate = LocalDateTime.now(),
            name = "name",
            statement = "statement",
            updateDate = LocalDateTime.now(),
            uuid = UUID.randomUUID().toString()
        )

        //Act
        val result = richSkillXlsExport.toXlsx(listOf(RichSkillAndCollections(rsd, setOf(collection))))

        //Assert
        Assertions.assertThat(result).isExactlyInstanceOf(ByteArray::class.java)
        Assertions.assertThat(result).isNotEmpty
    }
```

Reply:
@jchavez137 can I confirm that we are testing a return type of ByteArray, and that's all? Testing document builders is hard; it means build the document, save it, and crack it open to inspect / assert. What is the value gained from testing that this returns a ByteArray, when that's already the compiled return type?